tests/helpers.py

```diff
@@ -110,14 +112,17 @@ def create_secret(interface_type):
         secret_data = defaults.CSI_RBD_SECRET.copy()
         del secret_data['data']['kubernetes']
         secret_data['data']['admin'] = get_admin_key()
+        component = constants.RBD_COMPONENT
     elif interface_type == constants.CEPHFILESYSTEM:
         secret_data = defaults.CSI_CEPHFS_SECRET.copy()
         del secret_data['data']['userID']
         del secret_data['data']['userKey']
         secret_data['data']['adminID'] = constants.ADMIN_BASE64
         secret_data['data']['adminKey'] = get_admin_key()
+        component = constants.CEPHFS_COMPONENT
+
     secret_data['metadata']['name'] = create_unique_resource_name(
-        'test', 'secret'
+        f'test-{component}', 'secret'
```

Comment:
do we need the second arg 'secret'? I think we can remove this arg from create_unique_resource_name()

Reply:
The second arg 'secret' is    'resource_type' in create_unique_resource_name() which returns  return f"{resource_type}-{resource_description[:23]}-{current_date_time[:10]}"

If we remove second arg then, we can't identify which resource type are we creating

Reply:
yes, I meant that in create_uniques_resource_name, to return:
```python
return f"{resource_description[:23]}-{current_date_time[:10]}"
```

Reply:
Please disregard my above comment